Foundation/NSURLSession/http/HTTPURLProtocol.swift

```diff
@@ -127,6 +129,7 @@ fileprivate extension _HTTPURLProtocol {
                 set(requestBodyLength: .noBody)
             case (_, .some(let length)):
                 set(requestBodyLength: .length(length))
+                bodyLength = Int64(length)
```

Comment:
Sorry, ignore me, didn't quite realise this is the upload length rather than the Content-Length header sent by the server. Ignore me please!